yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/call/factory/SignatureFactoryTest.java

```diff
@@ -0,0 +1,32 @@
+package com.yoti.api.client.spi.remote.call.factory;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.security.KeyPair;
+
+import static com.yoti.api.client.spi.remote.util.CryptoUtil.KEY_PAIR_PEM;
+import static com.yoti.api.client.spi.remote.util.CryptoUtil.generateKeyPairFrom;
+import static com.yoti.api.client.spi.remote.util.CryptoUtil.verifyMessage;
+
+public class SignatureFactoryTest {
+
+    private static final String SOME_MESSAGE = "someMessage";
+
+    SignatureFactory testObj = new SignatureFactory();
+
+    KeyPair keyPair;
+
+    @Before
+    public void setUp() throws Exception {
+        keyPair = generateKeyPairFrom(KEY_PAIR_PEM);
+    }
+
+    @Test
+    public void shouldCreate() throws Exception {
```

Comment:
shouldCreateSignature?

Reply:
Can call it that, but it's a SignatureFactory.  The fact it's creating a signature should be clear.

This also changes in the next commit.